lib/module.js

```diff
@@ -635,7 +635,7 @@ Module._initPaths = function() {
 
   var nodePath = process.env['NODE_PATH'];
   if (nodePath) {
-    paths = nodePath.split(path.delimiter).filter(function(path) {
+    paths = nodePath.split(path.delimiter).filter(function pathsFilterCallback(path) {
```

Comment:
I agree, this is the only one that is needed. Can you please restore the other ones?